src/common/date.ts

```diff
@@ -26,3 +26,9 @@ export const isWeekend = (
       return day === DayOfWeek.Saturday || day === DayOfWeek.Sunday;
   }
 };
+
+export const getSecondsTimestamp = (ms: number | Date): number => {
```

Comment:
Isn't this just `debeziumTimeToDate`
https://github.com/dailydotdev/daily-api/blob/e20022838c55b6a5bd79bde372e6e15d7867bfc9/src/common/utils.ts#L138